pep-9999.rst

```diff
@@ -0,0 +1,293 @@
+beta releases would be amended to say "suitable for production use only in
+environments with sufficiently robust compatibility testing and operational
+monitoring capabilities".
+
+This PEP also proposes that the frequency of X.Y.0 releases be adjusted to
+target publishing a new release series in August of odd-numbered years
+(2021, 2023, etc).
+
+
+Background
+==========
+
+The current CPython pre-release and release management processes were developed
+in an era where automated continuous integration and operational monitoring
+systems were still relatively immature. Since that time, many organisations
+have adopted deployment models that allow them to incorporate new CPython
+feature releases without substantially more risk than they incur for any other
+code change.
+
+In light of that change in the wider environment, PEP 602 has proposed reducing
+the feature delivery latency for the Python standard library and CPython
+reference interpreter by increasing the frequency of CPython feature releases
+from every 18-24 months to instead occur every 12 months.
+
+Unfortunately, for many organisations, the cost of adopting a new Python release
+doesn't automatically scale down with a reduced number of changes in the release,
+as the primary costs aren't associated with resolving any discovered issues:
+the primary costs are associated with the *search* for issues. This search may
+involve manual testing of software systems, human review of written materials,
+and other activities where the time required scales with the size of the
+existing system, rather than the number of changes between the versions of
+Python.
+
+PEP 598 was an initial attempt by one of the authors of this PEP to propose
+an alternative scheme to reduce feature delivery latency by adopting a
+semantic versioning style policy that allowed for the incremental delivery of
+backwards compatible features within a release series until that series
+reached feature complete status. The major difference between that proposal
+and this one is to move the incremental feature delivery period into the
+pre-release phase of development (before strict API and ABI stability
+requirements apply), rather than having it apply to the initial X.Y.Z minor
+releases within a series.
+
+
+Example Future Release Schedules
+================================
+
+Under this proposal, Python 3.9.0b1 would be released in December 2019, two
+months after the Python 3.8.0 baseline feature release in October 2019.
+
+The 3.9.0b2 release would then follow 2 months later in February 2020,
+continuing through to 3.9.0b9 in April 2021.
+
+3.9.0rc1 would be published in June 2021, 3.9.0rc2 in July 2021, and then
+then the full release published as 3.9.0 in August 2021.
+
+The cycle would then begin over again in October 2021, with the publication
+of 3.10.0b1.
+
+Assuming maintenance releases of 3.9.x were also to occur every other month
```

Comment:
```suggestion
Assuming maintenance releases of 3.9.x were to occur every other month
```

Reply:
The "also" here refers to the beta release cadence, so I'll keep it for now.